sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -113,6 +110,46 @@ public virtual async Task<Response<TableItem>> CreateAsync(CancellationToken can
             }
         }
 
+        /// <summary>
+        /// Deletes a table in the storage account.
```

Comment:
suggestion: Deletes the current table.

Reply:
I also went ahead and changed the summary for `Create` and `CreateAsync` to match.